instrumentation/servlet/servlet-common/bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/servlet/ExperimentalSnippetHolder.java

```diff
@@ -5,12 +5,13 @@
 
 package io.opentelemetry.javaagent.bootstrap.servlet;
 
+import io.opentelemetry.instrumentation.api.internal.ConfigPropertiesUtil;
 import java.util.concurrent.atomic.AtomicReference;
 
 public class ExperimentalSnippetHolder {
 
   private static final AtomicReference<String> snippet =
-      new AtomicReference<>(System.getProperty("otel.experimental.javascript-snippet", ""));
+      new AtomicReference<>(ConfigPropertiesUtil.getString("otel.experimental.javascript-snippet"));
```

Comment:
`ConfigPropertiesUtil.getString()` may return `null`, which will break the `compareAndSet("", newValue)` call below.